retailchain.py

```
@@ -2,7 +2,7 @@
 import os
 from dotenv import load_dotenv
 from openai import OpenAI
-import anthropic
+import anthropi
```

Ruff (0.11.9)

`anthropi` imported but unused

Remove unused import: `anthropi`

(F401)